wicket-core/src/main/java/org/apache/wicket/coop/CoopConfiguration.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.wicket.coop;
+
+import org.apache.wicket.request.http.WebResponse;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public class CoopConfiguration
+{
+
+	enum CoopMode
```

Comment:
instead of overriding toString you can add a field to this enum 
```suggestion
	enum CoopMode {
	    UNSAFE_NONE("unsafe-none"),
	    SAME_SITE("same-site"),
	    SAME_ORIGIN("same-origin")

	    final String keyword;
	    CoopMode(String keyword){ this.keyword = keyword; }
	    
```